scripts/eda_kube.sh

```diff
@@ -227,8 +228,8 @@ case ${CMD} in
   "build") build-all "${VERSION}" ;;
   "clean") clean-deployment "${VERSION}";;
   "deploy") deploy "${VERSION}" ;;
-  "port-forward-api") port-forward-api 8000 ;;
-  "port-forward-ui") port-forward-ui 8080 ;;
+  "port-forward-api") port-forward-api ${PORT} ;;
+  "port-forward-ui") port-forward-ui ${PORT} ;;
```

Comment:
Should this more specific? e.g. `API_PORT` & `UI_PORT`

Reply:
you will run only one command at a time.

Reply:
What if you wanted to forward ports for both? Anyway you will have to do it in a new separate terminal but in the end I think explicit > implicit.

Reply:
the whole management of args is not a rebust implementation in this script I think, this is a small change to enable this, that's all. In fact I would like to remove it at some point in favor of ansible for various improvements for the minikube side.